File: Sirma Academy May 2024/JavaScript/Conditional Statements/FaceOfFigures.js
function calculateArea(figure, ...dimensions) {
  dimensions = dimensions.map(Number);

  if (figure === "square") {
    if (dimensions.length !== 1 || isNaN(dimensions[0])) {
      return "Invalid squire dimensions.";
    }
    return (dimensions[0] ** 2).toFixed(2);
  } else if (figure === "rectangle") {
    if (dimensions.length !== 2 || dimensions.some(isNaN)) {
      return "Invalid rectangle dimensions.";
    }
    return (dimensions[0] * dimensions[1]).toFixed(2);
  } else if (figure === "circle") {
    if (dimensions.length !== 1 || isNaN(dimensions[0])) {
      return "Invalid circle dimensions.";
    }
    return (dimensions[0] ** 2 * Math.PI).toFixed(2);
  } else if (figure === "triangle") {
    if (dimensions.length !== 2 || dimensions.some(isNaN)) {
      return "Invalid triangle dimensions.";
    }
    return ((dimensions[0] * dimensions[1]) / 2).toFixed(2);
  }
}

console.log(calculateArea("circle", 10));

File: Sirma Academy May 2024/Java/Conditional Statements/src/FaceOfFigures.java
import java.text.DecimalFormat;
import java.util.Scanner;

public class FaceOfFigures {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String figureType = sc.nextLine();
        DecimalFormat df = new DecimalFormat("#.##");
        double result = 0.0;

        switch (figureType.toLowerCase()){
            case "square":
                double sideLength = Double.parseDouble(sc.nextLine());
                result = sideLength * sideLength;
                break;
            case "rectangle":
                double sideX = Double.parseDouble(sc.nextLine());
                double sideY = Double.parseDouble(sc.nextLine());
                result = sideX * sideY;
                break;
            case "circle":
                double radius = Double.parseDouble(sc.nextLine());
                result = Math.PI * radius * radius;
                break;
            case "triangle":
                double side = Double.parseDouble(sc.nextLine());
                double height = Double.parseDouble(sc.nextLine());
                result = (side * height) / 2;
                break;
            default:
                System.out.println("Invalid figure type.");
        }
        System.out.printf("%.2f", result);

    }
}
